File: Solution in C++/Assignment_02/Task23.cpp
#include<iostream>
#include<iomanip>

using namespace std;

int main(){
    float f,c;
    cin>>f;
    c=(f-32)*0.56;
    cout<<setprecision(2)<<c<<" degrees C"<<endl;
    if (c<20){cout<<"Winter";}
    else if (c<=25){cout<<"Autumn";}
    else if(c<30){cout<<"Spring";}
    else{cout<<"Summer";}
}

File: Solution by Python/Assignment_02/Task23.py
F=float(input())
C=(F-32)*0.56
print(f"{int(C)} degrees C")
if C<20:
    print("Winter")
elif C>=20 and C<=25:
    print("Autumn")
elif C>25 and C<30:
    print("Spring")
else:
    print("Summer")